src/views/TechnicalAssistance/AdminHome/components/ReviewAdviceLetter/RecommendationsList.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, ButtonGroup } from '@trussworks/react-uswds';
+import classNames from 'classnames';
+
+import { GetTrbAdviceLetter_trbRequest_adviceLetter_recommendations as TRBRecommendation } from 'queries/types/GetTrbAdviceLetter';
+
+import RecommendationLinks from './RecommendationLinks';
+
+type RecommendationsListProps = {
+  type: 'form' | 'admin';
+  recommendations: TRBRecommendation[];
+  edit?: {
+    onClick: (recommendation: TRBRecommendation) => void;
+    text?: string;
+  };
+  remove?: {
+    onClick: (recommendation: TRBRecommendation) => void;
+    text?: string;
+  };
+  className?: string;
+};
+
+type CardTitleProps = {
+  type: RecommendationsListProps['type'];
+  children: React.ReactNode;
+  className?: string;
+};
+
+const CardTitle = ({ type, children, className }: CardTitleProps) => {
+  if (type === 'admin') {
+    return <h3 className={className}>{children}</h3>;
+  }
+  return <h4 className={className}>{children}</h4>;
+};
```

Comment:
This might be a little overkill for something that could be achieved by a ternary operator

```
{type === 'admin' ? <h3 className={className}>{children}</h3> : <h4 className={className}>{children}</h4>
```

Reply:
This is perfectly fine the way it is.